pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotSchemaRestletResource.java

```diff
@@ -207,7 +219,18 @@ public Representation put(Representation entity) {
   private Representation uploadSchema(
       @Parameter(name = "schemaName", in = "path", description = "The name of the schema to get")
       String schemaName) throws Exception {
-    File dataFile = getUploadContents();
+    File dataFile;
+    try {
+      dataFile = getUploadContents();
+    } catch (FileUploadBase.InvalidContentTypeException e) {
+      LOGGER.info("Invalid content type while updating schema {}", schemaName);
```

Comment:
as above but this one has schema name which is logged (we don't have schema name above)